src/libraries/System.Net.Http/tests/StressTests/HttpStress/ClientOperations.cs

```diff
@@ -42,7 +42,8 @@ public RequestContext(Configuration config, HttpClient httpClient, Random random
         public int TaskNum { get; }
         public bool IsCancellationRequested { get; private set; }
 
-        public Version HttpVersion => _config.HttpVersion;
+        public Version HttpVersion => _client.DefaultRequestVersion;
```

Comment:
Doesn't this work with the [`-http` config switch](https://github.com/dotnet/runtime/blob/91599fff26619cb8bccb7b39c39590e7cd9a4379/src/libraries/System.Net.Http/tests/StressTests/HttpStress/Program.cs#L82)?